name.abuchen.portfolio.tests/src/name/abuchen/portfolio/util/TLVTradeCalendarTest.java

```diff
@@ -0,0 +1,150 @@
+        
+        // YOM_KIPUR_DAY 
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-10-12")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-10-02")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-09-21")), is(true));
+
+
+        // FAST_DAY Tisha B'Av
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-08-13")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-08-03")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-07-23")), is(true));
+
+        // SIMCHAT_TORA_EVE, if (hebDay == 22 && hebMonth == 7)
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-10-23")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-10-13")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-10-02")), is(true));
+
+        // SIMCHAT_TORA; if (hebDay == 23 & hebMonth == 7)
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-10-24")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-10-14")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-10-03")), is(true));
+
+        // SAVHUOT_EVE, // SHAVUOT,
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-06-11")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-06-01")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-05-21")), is(true));
+
+        // SUKKOTH_EVE, if (hebDay == 14 && hebMonth == 7)
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-10-16")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-10-06")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-09-25")), is(true));
+
+        // SUKKOTH, if (hebDay == 15 && hebMonth == 7)
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-10-17")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-10-07")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-09-26")), is(true));
+    }
+
+    @Test
+    public void Special_Jewish_Calendar_Holidays_should_be_a_holiday()
+    {
+        // PURIM
+        TradeCalendar calendar = TradeCalendarManager.getInstance("tlv");
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-03-24")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-03-14")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-03-03")), is(true));
+
+    }
+
+    @Ignore
+    @Test
+    public void Israeli_Memorial_Day_should_be_a_holiday()
+    {
+        TradeCalendar calendar = TradeCalendarManager.getInstance("tlv");
+        assertThat(calendar.isHoliday(LocalDate.parse("2024-05-13")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2025-04-30")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2026-04-21")), is(true));
+    }
```

Comment:
What is the reason to "@Ignore" these tests?

I would prefer to either remove them (why test something that is not there?) or have a reason why this is ignored.